.github/workflows/build-dev.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
     - uses: actions/setup-go@v2
       with:
-        go-version: 1.16
+        go-version: 1.17
```

Comment:
Can you create a new PR to do golang upgrade?

Reply:
This PR need go1.17.